### PR TITLE
Fixed currency formatting

### PR DESCRIPTION
**Pull request by jepsar**

### CODE REVIEW

**Comment by melloware** [comment]
Do these changes work if you change the Locale to say Germany does the dollar switch to the euro?

I thought you could leave off `currencySymbol="$"` and it knows based on locale?

**Comment by jepsar** [comment]
Probably, but it is used like this one most places in the showcase. I just fixed a few exceptions.

**Comment by melloware** [comment]
I see!

File: src/main/webapp/ui/data/datatable/columnResize.xhtml [hunk 1]
```diff
@@ -48,7 +48,7 @@
             <div class="card">
                 <h5>Live Resize with Callback</h5>
                 <p:dataTable id="products2" var="product" value="#{dtResizableColumnsView.products2}"
-                             resizableColumns="true" liveResize="true" showGridlines="true">
```
Comment (jepsar) [inline src/main/webapp/ui/data/datatable/columnResize.xhtml:51]: Oops...

Comment (jepsar) [reply src/main/webapp/ui/data/datatable/columnResize.xhtml:51]: https://github.com/primefaces/primefaces-showcase/pull/351

**Pull request merged**